### PR TITLE
User invite client

### PR DESCRIPTION
**Pull request by sipsorcery**
Adds API client and methods for retrieving and sending user invites.

### CODE REVIEW

File: src/NoFrixion.MoneyMoov/Api/MerchantClient.cs [hunk 1]
```diff
@@ -146,4 +148,24 @@ public Task<MoneyMoovApiResponse<IEnumerable<PaymentAccount>>> GetAccountsAsync(
             _ => Task.FromResult(new MoneyMoovApiResponse<IEnumerable<PaymentAccount>>(HttpStatusCode.PreconditionFailed, new Uri(url), prob))
         };
     }
+
+    /// <summary>
+    /// Calls the MoneyMoov UserInvites Merchant endpoint to get the list of invites that have been
+    /// sent inviting new users to join the merchant.
+    /// </summary>
+    /// <param name="auserAccessToken">A User scoped JWT access token.</param>
```
Comment (Publicker) [inline src/NoFrixion.MoneyMoov/Api/MerchantClient.cs:156]: There's a typo in the param name, should be `userAccessToken`

File: src/NoFrixion.MoneyMoov/Api/UserClient.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using System.Net;
+
+namespace NoFrixion.MoneyMoov;
+
+public interface IUserClient
+{
+    //Task<MoneyMoovApiResponse<IEnumerable<PaymentAccount>>> GetAccountsAsync(string userAccessToken, Guid merchantID);
```
Comment (Publicker) [inline src/NoFrixion.MoneyMoov/Api/UserClient.cs:24]: Should this comment remain here?

**Review by Publicker — APPROVED** [review]

**Review by arifmatin — APPROVED** [review]

File: src/NoFrixion.MoneyMoov/Api/MerchantClient.cs (hunk 1 above)
Comment (sipsorcery) [reply src/NoFrixion.MoneyMoov/Api/MerchantClient.cs:156]: Thx, will fix.

File: src/NoFrixion.MoneyMoov/Api/UserClient.cs (hunk 2 above)
Comment (sipsorcery) [reply src/NoFrixion.MoneyMoov/Api/UserClient.cs:24]: Nope, will remove, thx.

**Pull request merged**